### DISCUSSION TITLE
Welcome to Orchard Core Discussions!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#6869] [General]
We're testing GitHub Discussions now, see https://github.com/OrchardCMS/OrchardCore/issues/6867 for related tasks. Stay tuned and please play around with it!

As we've talked about it before, GitHub has a new beta featured called Discussions. With Discussions, we can have Q&A and other non-issue discussions here on GitHub as well instead of the increasingly chaotic Gitter room. If somebody happens to remember, this is not unlike the Codeplex forum we had, plus Stack Overflow plus Github. This has the potential to greatly improve our community support, and thus ultimately make Orchard a better community and better project.

You can read about it in [this blogpost](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/) and [here's one of the first ones](https://github.com/vercel/next.js/discussions) for inspiration (there are no official docs about it yet). It's being continuously updated, one of the upcoming features being that a discussion won't necessarily need to be "answered" (that'll remain for Q&A-style ones only). I've asked for it and we got invited to the GitHub Discussions Limited Beta thus we can start using the feature.

A single person from every project can relay feedback and ask questions from GitHub under [a private project](https://github.com/gh-community/discussions-feedback), and that would be me. So let me know here (or let's discuss during the meetings) what we want to tell the GitHub team.

~~Discussions can have categories too but this is not enabled by default. I've asked the GitHub team to enable it for us.~~ Categories are now enabled. There are currently only these pre-configured ones but customizable categories are coming.

What do you think?

**Comment by MikeAlhayek** [comment]
This is so much better than using the Gitter chat! It'll make it easier to find answers for repeated questions.

Thank you!

**Comment by hishamco** [comment]
Finally ;) we should decide when to use discussions over GitHub issues

**Reply by Piedone** [reply]
While sometimes the line is blurred, I think only bug reports, feature requests, and other tasks that can be completed by changing something in Orchard, should go into the issue tracker. Everything else, here.

**Comment by gopkumr** [comment]
This is wonderful. Once there is forum like categories it would be easy to navigate (once the discussions gets too many). Definitely helpful for new members like me.

**Reply by Piedone** [reply]
Customizable categories are in the works, actually! For now, we'll have to do with the built-in ones.

**Comment by hishamco** [comment]
One more thing is there an option to filter or set milestones, coz I didn't see it

**Reply by Piedone** [reply]
Discussions are not related to milestones.

**Reply by hishamco** [reply]
Assume someone ask a question which leads to a bug, I think we need to track it in a certain milestone, so what we can do in that case?

**Reply by Piedone** [reply]
We can open an issue for it:
![image](https://user-images.githubusercontent.com/1976647/89901100-e76e1880-dbe4-11ea-8195-dc2ad9849b61.png)

**Comment by vitalybrandes** [comment]
Finaly!
thank you1
